projects/canopy/src/styles/styles.scss

```diff
@@ -7,3 +7,62 @@
 @import './typography';
 @import './utils';
 @import './variants';
+
+.lg-skeleton {
+  overflow: hidden;
+  position: relative;
+  display: flex;
+  flex-flow: column;
+  background-position: center;
+  background-size: 100% 100%;
+  background-repeat: no-repeat;
+  background-image: linear-gradient(
+    var(--color-platinum) calc(100% - 0.063em),
+    transparent 0
+  );
+  text-indent: -9999em;
+  max-width: 100%;
+}
+
+.lg-skeleton--right {
+  align-self: flex-end;
+  text-indent: 9999em;
+}
+
+.lg-skeleton:before {
+  content: '\200B';
+}
+
+.lg-skeleton::after {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  transform: translateX(-100%);
+  background-size: 100% 100%;
+  background-repeat: no-repeat;
+  background-color: transparent;
+  background-image: linear-gradient(
```

Comment:
@alexcanning have you ever looked at designing a colour/animation for a potential skeleton loading state before? This is just one I come up with for demo purposes but can be replaced with something more on-brand if available?